ami_spec.gemspec

```diff
@@ -17,11 +17,11 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.require_paths = ['lib']
 
-  gem.add_dependency 'aws-sdk-ec2'
+  gem.add_dependency 'aws-sdk-ec2', '~> 1'
   gem.add_dependency 'rake'
```

Comment:
As an aside, I wonder if `rake` is only used by ami-spec during development. In production it may not be needed.

If so we should mark it as a development dependency so downstream users of the gem aren't required to install Rake.

Reply:
🙇